functions/_fzf_search_pids.fish

```diff
@@ -0,0 +1,19 @@
+function _fzf_search_pids --description "Search pid for all running commands"
+    set fzf_arguments --multi --ansi $fzf_pids_opts
+    set token (commandline --current-token)
+    # expand any variables or leading tilde (~) in the token
+    #set expanded_token (eval echo -- $token)
+
+    set pid_selected (
+        ps -A -opid,command | \
```

Comment:
We need to filter out PID COMMAND headers from the output sent into fzf

Reply:
Added to the commit (--header-lines=1)